packages/consumption/src/modules/attributes/local/LocalAttribute.ts

```diff
@@ -131,6 +131,9 @@ export class LocalAttribute extends CoreSynchronizable implements ILocalAttribut
         if (typeof peerAddress !== "undefined") {
             isOwnSharedAttribute &&= this.shareInfo!.peer.equals(peerAddress);
         }
+
+        isOwnSharedAttribute &&= !this.hasPeerSharedAttributeDeletionInfo();
```

Comment:
why does the existence of a deletion info changes the type of an attribute?

Reply:
RepositoryAttributes are not allowed to have a deletionInfo, since it doesn't make sense. Own shared Attributes may only have the deletionStatus DeletedByPeer or ToBeDeletedByPeer, since DeletedByOwner and ToBeDeleted doesn't make sense and isn't implemented. Vice versa for peer shared Attributes.

Reply:
>since it doesn't make sense

By this you mean that we agreed to delete them immediately without any prior process to go through that would warrant a `deletionInfo` to keep track of any state, right?

Just making sure we're on the same boat.